public/TCP/TCPClient.js

```diff
@@ -27,6 +28,7 @@ function getConnectedClient() {
   // Handles receiving data
   client.on('data', function(data) {
     data = decodeData(data);
+    sendReceiveMessage();
```

Comment:
Not sure if this actually matters, but wouldn’t it make more sense to have this line after: 
```javascript
global.fromROV = data;
```

Reply:
Good point! Changed it now 👍